vars/buildXamarinIos.groovy

```diff
@@ -0,0 +1,35 @@
+def call(Map config) {
+
+    String solution = config.solution ?: findSolution()
+    if (solutionName == null) {
+    	failBuildWithError("No solution found. Please specify a solution explictly or ensure you are top level directory.")
+    }
+
+    String configuration = config.configuration ?: "Release"
+    String platform = config.platform ?: "iPhone"
```

Comment:
I assume this build step will work for macOS, tvOS apps as well? Can we maybe rename to reflect that? e.g. `buildXamarinApple` or something along those lines.

Reply:
yep but there's no Xamarin for tvOS so how about 'buildXamarinMacIos` ? Not a big fan of `buildXamarinApple`…

Reply:
I am of course wrong about no Xamarin for tvOS :)